deployment/api_transparency_dev/main.tf

```diff
@@ -0,0 +1,203 @@
+}
+resource "google_project_service" "storage_googleapis_com" {
+  service = "storage.googleapis.com"
+}
+
+module "lb-http" {
+  source  = "GoogleCloudPlatform/lb-http/google//modules/serverless_negs"
+  version = "~> 9.0"
+
+  name                  = var.lb_name
+  project               = var.project_id
+  load_balancing_scheme = "EXTERNAL_MANAGED"
+
+  ssl                             = var.tls
+  managed_ssl_certificate_domains = [var.serve_domain]
+  https_redirect                  = var.tls
+
+  create_ipv6_address = true
+  enable_ipv6         = true
+
+  backends = {
+    default = {
+      description = "Distributor API backend"
+      protocol    = "HTTPS"
+      port_name   = "https"
+      port        = 443
+      groups = [
+        {
+          group = google_compute_global_network_endpoint_group.distributor.id
+        }
+      ]
+
+      health_check = null
+
+      enable_cdn = false
+
+      iap_config = {
+        enable = false
+      }
+      log_config = {
+        enable = false
+      }
+    }
+  }
+
+  create_url_map = false
+  url_map        = google_compute_url_map.default.self_link
+}
+
+resource "google_compute_url_map" "default" {
+  name = "api-transparency-dev-url-map"
+
+  default_url_redirect {
+    https_redirect = true
+    host_redirect  = "transparency.dev"
+    path_redirect  = "/"
+    strip_query    = true
+  }
+
+  host_rule {
+    hosts        = [var.serve_domain]
+    path_matcher = "allpaths"
+  }
+
+  path_matcher {
+    name = "allpaths"
+
+    # If we don't know what this is, send them to the website.
+    default_url_redirect {
+      https_redirect = true
+      host_redirect  = "transparency.dev"
+      path_redirect  = "/"
+      strip_query    = true
+    }
+
+    #####
+    # Distributor rules
+
+    path_rule {
+      paths = [
+        "/distributor/*"
+      ]
+      route_action {
+        url_rewrite {
+          host_rewrite = var.distributor_host
+        }
+      }
+      service = module.lb-http.backend_services["default"].id
+    }
+
+    #####
+    # CI log & aretefacts rules
+    path_rule {
+      paths = [
+        "/armored-witness-firmware/ci/log/0/*"
+      ]
+      route_action {
+        url_rewrite {
+          path_prefix_rewrite = "/"
+        }
+      }
+      service = google_compute_backend_bucket.firmware_log_ci.id
+    }
+    path_rule {
+      paths = [
+        "/armored-witness-firmware/ci/artefacts/0/*"
+      ]
+      route_action {
+        url_rewrite {
+          path_prefix_rewrite = "/"
+        }
+      }
+      service = google_compute_backend_bucket.firmware_artefacts_ci.id
+    }
+
+    # TODO(prod logs & artefacts)
+  }
+}
+
+# Corresponding load balancer backend buckets.
+resource "google_compute_backend_bucket" "firmware_log_ci" {
+  name        = "firmware-log-ci-backend"
+  description = "Contains CI firmware transparency log"
+  bucket_name = "armored-witness-firmware-log-ci" # google_storage_bucket.armored_witness_firmware_log_ci.name
```

Comment:
Was this left here intentionally? (Below as well)

Reply:
Yup, I left it there as a breadcrumb/note for when we get around to adding and wiring up terraform outputs to the `build_and_release` unit.